app/assets/stylesheets/home.scss

```diff
@@ -1,14 +1,31 @@
-.curious {
-  background: asset-url("curiousity.jpg") no-repeat center center scroll;
-  background-size: cover;
+.columns {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+.about {
+  padding: 2rem;
+  background-color: #d5ffea;
+  border-radius: 50px;
+}
+
+.example-chart {
+  background-color: #d5ffea;
+  border-radius: 50px;
+
+  a {
+    color: black;
+    margin-top: 1rem;
+  }
+  a:hover {
+    color: #1585cf;
+  }
 }
 
-.curious h1 {
-  padding: 3rem;
-  color: white;
+div.module.text-center {
+  border-radius: 50px;
 }
 
-.curious button.button {
-  border-radius: 18px;
-  vertical-align: bottom;
+.learn {
+  background-color: #d5ffea;
```

Comment:
Color literals like `#d5ffea` should only be used in variable declarations; they should be referred to via variable everywhere else.